java_problem/sumofnumbers.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
```

Comment:
Please delete empty lines to make it nicer. Hint: don't do it from the website. Do it from your VM.

Reply:
Look into the git commit --amend command to perform this fix.